src/tests/ftest/io/ior_intercept_dfuse_mix.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+import threading
+import time
+import write_host_file
+from ior_test_base import IorTestBase
+from ior_utils import IorCommand, IorMetrics
+from command_utils import CommandFailure
+
+
+class IorInterceptDfuseMix(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR only with dfuse and with mix of
+       dfuse and interception library on a single server and multi
+       client settings with basic parameters.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        super(IorInterceptDfuseMix, self).setUp()
+        # Following if block can be removed once the constraint
+        # in IorTestBase is removed. # DAOS-3320
+        if self.ior_cmd.api.value == "POSIX":
+            self.hostlist_clients = self.params.get(
+                "test_clients", "/run/hosts/*")
```

Comment:
I don't think you need lines 48-50.
1. You are only using POSIX api
2. Regardless your "self.hostlist_clients" will still be picked automatically. Refer to apricot's test.py for more details.

Reply:
Will remove line 48 but I still need 49 and 50 because IorTestBase is limiting self.hostlist_clients to 1 if the api is POSIX.